### PR TITLE
docs: code splitting

### PR DESCRIPTION
**Pull request by DanDeMicco**

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @DanDeMicco has signed the CLA. Thanks for the pull request!

File: README.md [hunk 1]
```diff
@@ -296,6 +296,8 @@ render(
 );
 ```
 
+# Code Splitting
+[Code splitting](https://webpack.js.org/guides/code-splitting/) can be used with certain elements. As of now, the elements which support code splitting are `ContentPreview` and `ContentSidebar`. In order to use a code splitted element, you will need to build the element with webpack by importing the element from the `lib` folder in our npm package.
```
Comment (jstoffan) [inline README.md]: >[Code splitting](https://webpack.js.org/guides/code-splitting/) is currently support for the `ContentPreview` and `ContentSidebar` UI Elements. In order to use one of these Elements, ...

**Comment by jstoffan** [comment]
Seems like this change should be part of the main code splitting PR.

**Comment by DanDeMicco** [comment]
I wanted to keep this separate, as I wanted to create a separate PR for the `ContentSidebar` piece

File: README.md (hunk 1 above)
Comment (jeremypress) [reply README.md]: `In order to use an element with code splitting`?

**Comment by DanDeMicco** [comment]
Putting this into #737

**Pull request closed without merging**